src/Code.gs

```diff
@@ -38,21 +38,27 @@ function onInstall(e) {
  * @OnlyCurrentDoc
  */
 function addMenu() {
-  SpreadsheetApp.getUi().createAddonMenu()
+
+  var menu = SpreadsheetApp.getUi().createAddonMenu()
     .addItem('Re-Calculate all formulas in active sheet', 'recalcCustomFunctions')
     .addItem('Update Ticket Key Status "KEY-123 [Done]"', 'dialogRefreshTicketsIds')
     .addItem('Show Jira Field Map', 'sidebarJiraFieldMap')
     
     .addSeparator()
     .addItem('List Issues from Filter', 'dialogIssueFromFilter')
     .addItem('Create Time Report', 'dialogTimesheet')
-
+  
     .addSeparator()
     .addItem('Settings', 'dialogSettings')
     .addItem('Configure Custom Fields', 'dialogCustomFields')
-    .addItem('About', 'dialogAbout')
+    .addItem('About', 'dialogAbout');
+
+    if (environmentConfiguration.features.updateJira.enabled) {
```

Comment:
This will hide the option for this feature on production. That way we can commit this into develop and then into master without having to test the feature, just the potential side effects on existing features.